addons/portal/static/src/js/portal.js

```diff
@@ -282,6 +282,90 @@ publicWidget.registry.portalSecurity = publicWidget.Widget.extend({
 
 });
 
+/**
+ * Defining what happens when you click the "Log out from all devices" 
+ * on the "/my/security" page.
+ */
+publicWidget.registry.RevokeSessionsButton = publicWidget.Widget.extend({
+    selector: '#portal_revoke_all_sessions_popup',
+    events: {
+        click: '_onClick',
+    },
+
+    async _onClick(e) {
+        const rpc =this.proxy('_rpc')
```

Comment:
Various linting issues in this file, do you use a linter ? (spaces, missing ";", ...)